gqlalchemy/query_builder.py

```diff
@@ -146,15 +146,21 @@ def construct_query(self) -> str:
 
 
 class CallPartialQuery(PartialQuery):
-    def __init__(self, procedure: str, arguments: str):
+    def __init__(self, procedure: str, arguments: Optional[Union[str, Tuple[Union[str, int, float]]]]):
         super().__init__(DeclarativeBaseTypes.CALL)
 
         self.procedure = procedure
-        self.arguments = arguments
+        self.arguments = self._parse_query_arguments(arguments)
 
     def construct_query(self) -> str:
         return f" CALL {self.procedure}({self.arguments if self.arguments else ''}) "
 
+    def _parse_query_arguments(self, arguments: Optional[Union[str, Tuple[Union[str, int, float]]]]) -> str:
```

Comment:
Similar methods were called `to_cypher_<>`, e.g., `to_cypher_qm_arguments()`, and located in [utilities](https://github.com/memgraph/gqlalchemy/blob/develop/gqlalchemy/utilities.py#L40). Let me know what you think, this is just a suggestion.